lib/LaTeXML/resources/CSS/LaTeXML.css

```diff
@@ -88,7 +88,8 @@ a.ltx_LaTeXML_logo { text-decoration: none; }
 /*======================================================================
   Para level */
 .ltx_float {
-    margin: 1ex 3em 1ex 3em; }
+    margin: 1ex 3em 1ex 3em;
+    margin: 1ex max(1em,min(3em,4.5%)) 1ex max(1em, min(3em, 4.5%)); }
```

Comment:
I had to blink a few times but I think I'm seeing this right - shouldn't the first of each repeating rule be removed? as in:
```
margin: ...
margin: ...
```

and in a few rules further down:
```
margin-left: ...
margin-left: ...
```

I think you only need to keep the last of each of those, right?

Reply:
...part of me still tries to support IE...

I have always wondered, does LaTeXML have a target in mind? E.g. anything that has been supported for at least 3 years in major browsers (just making one up)? Or something based on caniuse.com statistics?

Reply:
Regarding margins versus width: If the goal is to keep from having unreadably wide paragraphs on wide displays, I'm inclined to think that we would end up with a more adaptable & tweakable system if we think of margins for things like abstracts, lists, tables, etc, that appear "within the flow".  But only restrict the overall width at higher level.   Then (perhaps), you can adjust overall page width or abstract "width" by making fewer changes, without having to track down as many surprises.

Obviously, there are many ways of getting the same look for any combination of page, browser, etc, but It can be simpler the fewer knobs there are that do the Right Thing.

Reply:
> Then (perhaps), you can adjust overall page width or abstract "width" by making fewer changes, without having to track down as many surprises.

Should I add a max-width to body, while I am here? Something like @dginev's `52em`?

Reply:
@xworld21 I would add max-width in passing, since a variety of other bits will no longer work as intended (in particular I think it will harm the flexbox subfigures with the default CSS).

Best to do that separately, if at all.